src/lib/reporters/Html.ts

```diff
@@ -3,6 +3,32 @@ import Reporter, { eventHandler, ReporterProperties } from './Reporter';
 import Test from '../Test';
 import Suite from '../Suite';
 
+const location = window.location;
+
+function getFullName(test: Suite | Test): string {
```

Comment:
Don't specify a return type when it's not necessary.

Reply:
done